tesseract/tesseract_planning/tesseract_motion_planners/include/tesseract_motion_planners/core/planner.h

```diff
@@ -31,25 +31,33 @@ TESSERACT_COMMON_IGNORE_WARNINGS_PUSH
 #include <unordered_map>
 TESSERACT_COMMON_IGNORE_WARNINGS_POP
 
-#include <tesseract_planners/core/types.h>
+#include <tesseract_motion_planners/core/types.h>
 
-namespace tesseract_planners
+namespace tesseract_motion_planners
 {
-class BasicPlanner
+class MotionPlanner
 {
 public:
   /** @brief Construct a basic planner */
-  BasicPlanner() {}
-  virtual ~BasicPlanner() {}
+  MotionPlanner() {}
+  virtual ~MotionPlanner() {}
   /** @brief Get the name of this planner */
   const std::string& getName() const { return name_; }
   /** @brief Get the planner request */
   const PlannerRequest& getRequest() const { return request_; }
   /** @brief Set the planner request for this context */
   void setRequest(const PlannerRequest& request) { request_ = request; }
+
   const StatusCodeMap& getAvailableStatusCodes() const { return status_code_map_; }
+
   /** @brief Solve the planner request problem */
-  virtual bool solve(PlannerResponse& res) = 0;
+  virtual bool solve(PlannerResponse& res) const = 0;
```

Comment:
I not sure this should be const. Some planners may want cache data from request to request like a road map planner. What do you think?

Reply:
OK, that's a used case I hadn't thought of, I'll change it to non-const